pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.466</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>1.509</version><!-- which version of Jenkins is this plugin built against? -->

```

Comment:
Script security requires 1.509.
(Though script-security-1.9 is built for Jenkins 1.466, script-security is basically designed for Jenkins >= 1.509)